tests/database/migrations/2020_10_02_043202_create_sessions_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+final class CreateSessionsTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('sessions', function (Blueprint $table) {
```

Comment:
doesn't this come with laravel by default? https://laravel.com/docs/8.x/session#database

Reply:
I thought so too, but it's not (anymore): https://github.com/laravel/laravel/tree/8.x/database/migrations

I needed this migration for tests.